### PR TITLE
tooling + actions pr

### PR DESCRIPTION
**Pull request by zeeshanlakhani**
## Summary

Tooling PR, including:

* cleaned up ci / actions for consistency w/ rs-ucan
* move to minor versioned deps, and fix jumpsize need w/ update to 0.1.2 of rs-skip-ratchet
* remove cargo.lock top-level
* remove wee_alloc due to sec vulns, as it's outdated.

### CODE REVIEW

**Comment by appcypher** [comment]
Awesome @zeeshanlakhani! It is great to see the security CI checks. 🎉

File: crates/wasm/Cargo.toml [hunk 1]
```diff
@@ -17,19 +17,18 @@ homepage = "https://fission.codes"
...
-wasm-bindgen-futures = { version = "0.4", optional = true }
-js-sys = { version = "0.3", optional = true }
-web-sys = { version = "0.3", optional = true }
-chrono = { version = "0.4", features = ["wasmbind"] }
 anyhow = "1.0"
 async-trait = "0.1"
+cfg-if = "1.0"
+chrono = { version = "0.4", default-features = false, features = ["clock", "std", "wasmbind"] }
 console_error_panic_hook = { version = "0.1", optional = true }
-wee_alloc = { version = "0.4", optional = true }
```
Comment (appcypher) [inline crates/wasm/Cargo.toml:30]: Turns out wee-alloc hasn't brought much improvement to our wasm size or we've been using it wrong.

Comment (zeeshanlakhani) [reply crates/wasm/Cargo.toml:30]: It's essentially dead anyway. https://github.com/rustwasm/wee_alloc/issues/85 + https://github.com/rustsec/advisory-db/issues/1394. We can look at other allocators tho, of interest.

**Comment by zeeshanlakhani** [comment]
still working on the move to grcov in ci.

File: README.md [hunk 2]
```diff
@@ -158,32 +158,36 @@ async fn main() {
     cargo install wasm-pack
     ```
 
-  - Install playwrigth binaries
+  - Install [playwright](https://www.npmjs.com/package/playwright) binaries
```
Comment (appcypher) [inline README.md:161]: Is it better to link to the site?

```suggestion
  - Install [playwright](https://playwright.dev/) binaries
```

**Comment by zeeshanlakhani** [comment]
ok, coverage updated minus some codecov UI weirdness.

**Review by appcypher — APPROVED** [review]
<div align="center">
    <a href="#" target="_blank">
        </img>
    </a>
</div>

<h3 align="center">Looks good to me! 🎉 </h3>

**Review by matheus23 — APPROVED** [review]
:tada:

**Pull request merged**